editor/script_create_dialog.cpp

```diff
@@ -128,8 +129,23 @@ void ScriptCreateDialog::_notification(int p_what) {
 		} break;
 
 		case NOTIFICATION_THEME_CHANGED: {
+			const int icon_size = get_theme_constant(SNAME("class_icon_size"), EditorStringName(Editor));
+
+			EditorData &ed = EditorNode::get_editor_data();
+
 			for (int i = 0; i < ScriptServer::get_language_count(); i++) {
-				Ref<Texture2D> language_icon = get_editor_theme_icon(ScriptServer::get_language(i)->get_type());
+				// Check if the extension has an icon first
+				String script_type = ScriptServer::get_language(i)->get_type();
```

Comment:
This can be inside the condition.

Reply:
It can? It's used by `get_editor_theme_icon`....

Reply:
Right...